### ISSUE TITLE
linter: `eslint-plugin-import(namespace)` gives false warning for TypeScript enum

### ISSUE THREAD

**Issue by timostamm** [oxc-project/oxc#10318]
### What version of Oxlint are you using?

0.16.5

### What command did you run?

`oxlint`

### What does your `.oxlintrc.json` config file look like?

```jsonc
{
  "$schema": "./node_modules/oxlint/configuration_schema.json",
  "plugins": ["import"],
  "rules": {
    "import/namespace": "error"
  }
}
```

### What happened?

The rule is triggered:

```
  × eslint-plugin-import(namespace): "Foo" not found in imported namespace "./lib".
   ╭─[test.ts:3:24]
 2 │ 
 3 │ export const bar = lib.Foo.BAR;
```

However, `Foo` is defined in `lib.ts`:

```
export enum Foo {
  BAR = 0
}
```

Here's a full repro: https://github.com/timostamm/oxlint-import-namespace-enum

**Comment by ulrichstark** [comment]
Fixed in oxlint@0.16.8
